### PR TITLE
Intake Stats page MVP

### PR DESCRIPTION
**Pull request by shanear**
MVP skeleton of the Intake stats page with some easy to calculate metrics.

To test, visit `/intake/stats`

# Notes
Right now you have to recalculate stats by using `Rails.cache.clear` from the console, or if you concerned with updating stats, just run `IntakeStats.calculate_all!`

The plan is to add a job in the future that does this automagically.

connects #3765

### CODE REVIEW

File: app/assets/images/caseflow-intake-logo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...3.34,34.54,28.75-12.73-12.79-33.13a20.89,20.89,0,0,1,0-15.08l12.78-33.1-28.75-12.73m-88.2-104.2-33.16,12.81a21.09,21.09,0,0,1-15.09,0l-33-12.74-12.73,28.75L231,134.45a52.57,52.57,0,0,0,37.71,0l34.59-13.35L290.61,92.34M110,57.81a52.38,52.38,0,0,0-43.1,23A51.49,51.49,0,0,0,61.4,129.1L81,180l28.75-12.73L90.7,117.79a20.25,20.25,0,0,1,2.14-19.25,20.82,20.82,0,0,1,24.93-7.83l49.51,19.12L180,81.08,129.09,61.41a52.8,52.8,0,0,0-19-3.6m279.86-.06a52.78,52.78,0,0,0-19,3.6L319.94,81l12.73,28.75,49.49-19.11a20.82,20.82,0,0,1,24.93,7.83,20.25,20.25,0,0,1,2.14,19.24l-19.11,49.48,28.75,12.73L438.54,129A51.48,51.48,0,0,0,433,80.72a52.38,52.38,0,0,0-43.1-23" transform="translate(0 0)"/><path class="cls-3" d="M303.36,378.85l-12.73,28.75L320,418.93l12.73-28.75-29.33-11.33m-106.77-.06-29.34,11.33L180,418.87l29.34-11.33L196.6,378.79m211-88.21-28.75,12.73,11.33,29.33,28.75-12.73-11.33-29.33m-315.21,0L81,319.88l28.75,12.73,11.33-29.33L92.36,290.55m17.44-123.31L81,180,92.37,209.3l28.75-12.73-11.33-29.33m280.34,0-11.33,29.33,28.75,12.73,11.33-29.33-28.75-12.73M180,81.08l-12.73,28.75,29.33,11.33L209.36,92.4,180,81.08M319.94,81,290.61,92.34l12.73,28.75,29.33-11.33L319.94,81" transform="translate(0 0)"/></svg>
```
Comment (shanear) [inline app/assets/images/caseflow-intake-logo.svg]: The stats pages still use the good ole fashioned logos. Not going to try and change this for now

File: app/controllers/intake_stats_controller.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      monthly: 0...24
+    }[interval].map { |i| IntakeStats.offset(time: IntakeStats.now, interval: interval, offset: i) }
+  end
+
+  def logo_name
+    "Intake"
+  end
+
+  def interval
+    @interval ||= IntakeStats::INTERVALS.find { |i| i.to_s == params[:interval] } || :monthly
```
Comment (shanear) [inline app/controllers/intake_stats_controller.rb]: Default to monthly, since that is the most useful for AMO

File: app/models/intake_stats.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      RampElection.where(notice_date: offset_range(range)).count
+    end,
+
+    # Number of opt-in elections received by month and FYTD
+    elections_successfully_received: lambda do |range|
+      RampElection.completed.where(receipt_date: offset_range(range)).count
+    end,
+
+    # Average days to respond to RAMP election notice
+    average_election_response_time: lambda do |range|
```
Comment (cmgiven) [inline app/models/intake_stats.rb]: This is should be `median_election_response_time`.

File: app/views/intake_stats/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+          Successfully Received
+        </h4>
+        <div class="cf-stat-figure data-value" data-key="elections_successfully_received">
+          <%= @stats[0].values[:elections_successfully_received] %>
+        </div>
+      </div>
+
+      <div class="cf-stat-panel data-chart" data-key="average_election_response_time">
+        <h4 class="cf-stat-title">
+          Average Response Time
```
Comment (cmgiven) [inline app/views/intake_stats/show.html.erb]: Median Response Time.

File: app/controllers/intake_stats_controller.rb (hunk 2 above)
Comment (cmgiven) [reply app/controllers/intake_stats_controller.rb]: Agreed this makes the most sense.

**Comment by cmgiven** [comment]
It occurs to me that they might also want an uptake rate, as in of the letters sent out during a given time, how many have responded with an election?

File: app/views/intake_stats/show.html.erb (hunk 4 above)
Comment (shanear) [reply app/views/intake_stats/show.html.erb]: Thank you stats god 🙇

**Comment by shanear** [comment]
> It occurs to me that they might also want an uptake rate, as in of the letters sent out during a given time, how many have responded with an election?

Yeah, this might be on the radar for a future addition

**Pull request merged**